### PR TITLE
Min 340

### PR DESCRIPTION
**Pull request by Winson-Lam-acer-org**

### CODE REVIEW

File: shr-boot/action.yml [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+    required: false
+  runners_data_file:
+    description: 'where to get the csv file for runners data'
+    required: false
+
+runs:
+  using: "composite"
+  steps:
+    - id: settings
+      uses: acerorg/gha-runner/get-settings@MIN-340
```
Comment (Winson-Lam-acer-org) [inline shr-boot/action.yml:21]: once merged into `main` branch, I will need to change these to point to `main` as well.

File: shr-remove/action.yml [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+    required: false
+  runners_data_file:
+    description: 'where to get the csv file for runners data'
+    required: false
+
+runs:
+  using: "composite"
+  steps:
+    - id: settings
+      uses: acerorg/gha-runner/get-settings@MIN-340
```
Comment (Winson-Lam-acer-org) [inline shr-remove/action.yml:21]: once merged into `main` branch, I will need to change these to point to `main` as well.

File: shr-remove/action.yml [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+      with:
+        scope: ${{ inputs.scope }}
+        token: ${{ inputs.token }}
+        runners_data_file: ${{ inputs.runners_data_file }}
+
+    - uses: acerorg/gha-runner/use-python@MIN-340
```
Comment (Winson-Lam-acer-org) [inline shr-remove/action.yml:27]: once merged into `main` branch, I will need to change these to point to `main` as well.

File: shr-remove/action.yml [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+    - uses: actions/checkout@v3
+
+    - name: Get runner details
+      id: get-runner-details
+      uses: acerorg/gha-runner/get-runner-details-from-csv@MIN-340
```
Comment (Winson-Lam-acer-org) [inline shr-remove/action.yml:32]: once merged into `main` branch, I will need to change these to point to `main` as well.

**Comment by Winson-Lam-acer-org** [comment]
once merged into `main` branch, I will need to change these to point to `main` as well.
mainly within the `shr-*/action.yml` files

**Review by June-Lin-acer-org — APPROVED** [review]

**Pull request merged**